### DISCUSSION TITLE
How can I use vue-router?

### DISCUSSION THREAD

**Discussion by rubie0714** [FranckFreiburger/vue3-sfc-loader#49] [Q&A] [answered]
I change my project form httpVueLoader to vue3-sfc-loader and Convert from Vue 2 to Vue 3
I use example code and add  vue-router cdn script tag I get this error 
Uncaught TypeError: window.Vue.use is not a function
Do I need to add anything else?

**Comment by FranckFreiburger** [comment, marked as answer]
Hi,
Vue3 has changed the way to create a new application (see https://v3.vuejs.org/guide/instance.html)
You can also have a look at https://github.com/FranckFreiburger/vue3-sfc-loader/issues/43

**Reply by rubie0714** [reply]
I got it and Thanks for your reply

**Comment by FranckFreiburger** [comment]
see. https://github.com/FranckFreiburger/vue3-sfc-loader/discussions/172
